litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/styles.ts

```diff
@@ -262,6 +258,35 @@ const useStyles = makeStyles((theme) => ({
     width: '0.625rem',
     height: '0.625rem',
   },
+
+  export: {
+    verticalAlign: 'middle',
+    display: 'inline-flex',
+    textAlign: 'center',
+    borderRadius: 3,
```

Comment:
Can you convert it into rem?

Reply:
`borderRadius` is very small to be used in rem, it will be 3/16. At all other places, we have used `3` in the absolute unit for this border-radius. @Saranya-jena